python/lsst/cp/pipe/ptc.py

```diff
@@ -889,14 +914,17 @@ def _getInitialGoodPoints(means, variances, maxDeviationPositive, maxDeviationNe
         goodPoints = np.array([True if (r < maxDeviationPositive and r > maxDeviationNegative)
                               else False for r in ratioDeviations])
 
-        # Discard points when variance starts decreasing
+        # Eliminate points beyond which the variance decreases
         pivot = np.where(np.array(np.diff(variances)) < 0)[0]
-        if len(pivot) == 0:
-            return goodPoints
-        else:
-            pivot = np.min(pivot)
-            goodPoints[pivot:len(goodPoints)] = False
-            return goodPoints
+        if len(pivot) > 0:
+            # For small values, sometimes the variance decreases slightly
+            # Only look when var > self.config.minVarPivotSearch
+            pivot = [p for p in pivot if variances[p] > minVarPivotSearch]
```

Comment:
Just confirming that applying the minimum in the variance space is correct.  Cutting on the mean seems better to me, as this is a saturation effect, and we want to not cut for saturation before we think we're at that point.  This also feels odd because we're looking for a turnover but only if the turnover is above that minimum value.  I'm fine either way, so feel free to ignore this.

Reply:
"This also feels odd because we're looking for a turnover but only if the turnover is above that minimum value"---> Right, this is what this `minVarPivotSearch` cut is meant to help with. 
In addition, at this point, the mean vector has been sorted, and the variance vector too using the indices of the mean vector. The code at this point assumes that the variance, after being sorted by the index of the sorted mean, increases monotonically.